internal/services/mssql/mssql_server_dns_alias_resource.go

```diff
@@ -0,0 +1,143 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/azure"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/sdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/mssql/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/mssql/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ServerDNSAliasModel struct {
+	ServerName    string `tfschema:"server_name"`
+	ResourceGroup string `tfschema:"resource_group_name"`
+	//Kind          string `tfschema:"kind"`
+	DNSAliasName   string `tfschema:"dns_alias_name"`
+	AzureDNSRecord string `tfschema:"azure_dns_record"`
```

Comment:
This can be shorten into `DNSRecord` (`dns_record`).

Reply:
these field names are same as the API definition, should i change it to a short name?

Reply:
Yes, please